caffe2/ideep/utils/ideep_register.cc

```diff
@@ -1,7 +1,7 @@
 #include <caffe2/core/event_cpu.h>
 #include <caffe2/core/operator.h>
 #include <caffe2/proto/caffe2_pb.h>
-#include <ideep_pin_singletons.hpp>
+#include <ideep.hpp>
```

Comment:
`ideep/tensor.hpp` should be enough since only ideep::tensor has been used in this file